### PR TITLE
EndpointElem -> VPCResourceIntf

### PR DESCRIPTION
**Pull request by ShiriMoran**

### CODE REVIEW

**Review by adisos — DISMISSED** [review]

File: pkg/ibmvpc/analysis_output_test.go
Comment (adisos) [inline pkg/ibmvpc/analysis_output_test.go:1]: Why is this file changed on this PR?

Comment (ShiriMoran) [reply pkg/ibmvpc/analysis_output_test.go:1]: no idea

Comment (ShiriMoran) [reply pkg/ibmvpc/analysis_output_test.go:1]: problem that occurred during merge. fixed

**Review by adisos — APPROVED** [review]
